lein-environ/test/lein_environ/plugin_test.clj

```diff
@@ -0,0 +1,15 @@
+(ns lein-environ.plugin-test
+  (:require [clojure.test :refer :all]
+            [lein-environ.plugin :as l]))
+
+(deftest replace-project-keyword-test
+  (are [expected project] (= expected (l/read-env project))
+    {:app-version  "1.0.0"
+     :test-version :version}
+    {:version "1.0.0"
+     :env     {:app-version  :project/version
+               :test-version :version}}
+
+    {1 1} {:env {1 1}}
+    {:tests {:repl? false}} {:tests {:repl? false}
+                    :env {:tests :project/tests}}))
```

Comment:
Can you fix the indentation here? Or just to make it consistent, maybe do:

```clojure
(are [expected project] (= expected (l/read-env project))
  {:app-version "1.0.0", :test-version :version}
  {:version "1.0.0", :env {:app-version :project/version :test-version :version}}

  {:foo "bar"}
  {:env {:foo "bar"}})
```

Note that I've also removed/changed invalid environments. The env map must consist of keywords to strings (which is checked by `environ.core`).